# name: error output 
python hello.py 2> "error.err"